.gitignore

```diff
@@ -11,3 +11,4 @@ osx-utils/Performous.dmg
 osx-utils/CMakeCache.txt
 CMakeCache.txt
 CMakeFiles
+*.xcodeproj
```

Comment:
does this need to be in the dmg package?

Reply:
No; I don't usually even use xcode for coding, but I wanted to import the performous repo into it because it's re-indent feature is frankly really, really good.